tests/src/test/java/com/netflix/msl/msg/MessageBuilderSuite.java

```diff
@@ -971,13 +1037,16 @@ public void deletePeerServiceToken() throws MslEncodingException, MslCryptoExcep
         public void deleteUnknownPeerServiceToken() throws MslException {
             final MessageBuilder builder = messageFactory.createRequest(p2pCtx, MASTER_TOKEN, USER_ID_TOKEN);
             builder.setPeerAuthTokens(PEER_MASTER_TOKEN, PEER_USER_ID_TOKEN);
-            builder.deletePeerServiceToken(SERVICE_TOKEN_NAME);
+            builder.deletePeerServiceToken(SERVICE_TOKEN_NAME, true, true);
             final MessageHeader messageHeader = builder.getHeader();
             final Set<ServiceToken> tokens = messageHeader.getPeerServiceTokens();
             for (final ServiceToken token : tokens) {
-                if (token.getName().equals(SERVICE_TOKEN_NAME))
-                    fail("Deleted unknown peer service token.");
+                if (token.getName().equals(SERVICE_TOKEN_NAME)) {
+                    assertEquals(0, token.getData().length);
```

Comment:
same

Reply:
Same as above length == 0 confirms the "deletion token" was created properly.